src/views/GovernanceReviewTeam/Decision/index.tsx

```diff
@@ -0,0 +1,155 @@
+            <DescriptionDefinition
+              className="text-pre-wrap"
+              definition={systemIntake.lcid}
+            />
+          </div>
+        </ReviewRow>
+        <ReviewRow>
+          <div>
+            <DescriptionTerm
+              term={t('governanceReviewTeam:decision.lcidExpiration')}
+            />
+            <DescriptionDefinition
+              definition={
+                systemIntake.lcidExpiration
+                  ? systemIntake.lcidExpiration.toLocaleString(
+                      DateTime.DATE_FULL
+                    )
+                  : ''
+              }
+            />
+          </div>
+        </ReviewRow>
+        <ReviewRow>
+          <div>
+            <DescriptionTerm term={t('governanceReviewTeam:decision.scope')} />
+            <DescriptionDefinition
+              className="text-pre-wrap"
+              definition={systemIntake.lcidScope}
+            />
+          </div>
+        </ReviewRow>
+        <ReviewRow>
+          <div>
+            <DescriptionTerm
+              term={t('governanceReviewTeam:decision.nextSteps')}
+            />
+            <DescriptionDefinition
+              className="text-pre-wrap"
+              definition={systemIntake.decisionNextSteps}
+            />
+          </div>
+        </ReviewRow>
+      </DescriptionList>
+    </>
+  );
+
+  const Rejected = () => (
+    <>
+      <h1>{t('governanceReviewTeam:decision.titleRejected')}</h1>
+      <DescriptionList
+        title={t('governanceReviewTeam:decision.decisionSectionTitle')}
+      >
+        <ReviewRow>
+          <div>
+            <DescriptionTerm
+              term={t('governanceReviewTeam:decision.rejectionReason')}
+            />
+            <DescriptionDefinition
+              className="text-pre-wrap"
+              definition={systemIntake.rejectionReason}
+            />
+          </div>
+        </ReviewRow>
+        <ReviewRow>
+          <div>
+            <DescriptionTerm
+              term={t('governanceReviewTeam:decision.nextSteps')}
+            />
+            <DescriptionDefinition
+              className="text-pre-wrap"
+              definition={systemIntake.decisionNextSteps}
+            />
+          </div>
+        </ReviewRow>
+      </DescriptionList>
+    </>
+  );
+
+  const NotItRequest = () => (
+    <>
+      <h1>{t('governanceReviewTeam:decision.titleClosed')}</h1>
+      <p>{t('governanceReviewTeam:decision.descriptionNotItRequest')}</p>
+    </>
+  );
+
+  const NoGovernance = () => (
+    <>
+      <h1>{t('governanceReviewTeam:decision.titleClosed')}</h1>
+      <p>{t('governanceReviewTeam:decision.descriptionNoGovernance')}</p>
+    </>
+  );
+
+  const ShutdownComplete = () => (
+    <>
+      <h1>{t('governanceReviewTeam:decision.titleClosed')}</h1>
+      <p>{t('governanceReviewTeam:decision.shutdownComplete')}</p>
+    </>
+  );
+
+  if (systemIntake.status === 'LCID_ISSUED') {
```

Comment:
Given there's so many cases, I think a `switch` statement might be easier to read, but no strong opinions.